scrapi/base/__init__.py

```diff
@@ -30,17 +31,25 @@ class BaseHarvester(object):
 
     @abc.abstractmethod
     def harvest(self, days_back=1):
-        pass
+        raise NotImplementedError
```

Comment:
I don't think you need these. I suppose it's explicit, but I don't think we'll ever get there.